vendor/github.com/moolitayer/awx-client-go/awx/job_template_launch_resource.go

```diff
@@ -76,18 +78,24 @@ func (r *JobTemplateLaunchGetResponse) JobTemplateData() *JobTemplate {
 type JobTemplateLaunchPostRequest struct {
 	Request
 
-	extraVars string
+	extraVars map[string]interface{}
 }
 
-func (r *JobTemplateLaunchPostRequest) ExtraVars(value string) *JobTemplateLaunchPostRequest {
+func (r *JobTemplateLaunchPostRequest) ExtraVars(value map[string]interface{}) *JobTemplateLaunchPostRequest {
```

Comment:
I think that in addition to this `ExtraVars` method we should also have another `ExtraVar` method that receives only the variable name and the value, so that users of the AWX client can do something like this:

```go
response := resource.NewJobTemplateLaunch().
  ExtraVar("myvar", "myvalue").
  ExtraVar("yourvar", 123).
  ...
  Send()
```